api/tests/test_user.py

```diff
@@ -238,14 +238,48 @@ def test_add_user_api_endpoint_cant_allow_put(self, mock_verify_token):
         self.assertEqual(response.json(), {'detail': 'Method "PUT" not allowed.'})
 
     @patch('api.authentication.auth.verify_id_token')
-    def test_user_api_endpoint_cant_allow_patch(self, mock_verify_token):
+    def test_user_api_endpoint_can_allow_patch(self, mock_verify_token):
         mock_verify_token.return_value = {'email': self.admin_user.email}
         user = User.objects.first()
         response = client.patch(
             '{}/{}/'.format(self.users_url, user.id),
             HTTP_AUTHORIZATION="Token {}".format(self.token_admin),
         )
-        self.assertEqual(response.json(), {'detail': 'Method "PATCH" not allowed.'})
+        self.assertTrue(response.status_code == 200)
+
+    @patch('api.authentication.auth.verify_id_token')
+    def test_admin_user_api_endpoint_allow_patch_for_is_staff_attr(
+        self, mock_verify_token
+    ):
+        mock_verify_token.return_value = {'email': self.admin_user.email}
+        user = User.objects.first()
+        update_data = {'is_staff': True}
+        response = client.patch(
+            '{}/{}/'.format(self.users_url, user.id),
+            data=update_data,
+            HTTP_AUTHORIZATION="Token {}".format(self.token_admin),
+        )
+        self.assertEqual(response.status_code, 200)
+        is_staff = User.objects.first().is_staff
```

Comment:
would be safer to fetch user by id. please also add an assertion to check that other fields haven't changed.

Reply:
Adding check that other fields haven't changed would be me testing that the patch function works well; Django has test for that already. Patch updates the attributes that are passed via request.